ui/app/templates/components/clients/dashboard.hbs

```diff
@@ -70,162 +50,148 @@
           to enable tracking again.
         </AlertBanner>
       {{/if}}
-      <h1 data-test-client-count-title class="title is-4 has-bottom-margin-s">
-        Monthly history
-        {{! ARG TODO change }}
-      </h1>
-      <p class="has-bottom-margin-s">
-        This data is presented by full month. If there is data missing, it's possible that tracking was turned off at the
-        time. Vault will only show data for contiguous blocks of time during which tracking was on.
-      </p>
-      {{! TODO remove once you remove the monthly view }}
-      <PricingMetricsDates
-        @queryStart={{@model.queryStart}}
-        @queryEnd={{@model.queryEnd}}
-        @resultStart={{@model.activity.startTime}}
-        @resultEnd={{@model.activity.endTime}}
-        @defaultSpan={{or @model.config.defaultReportMonths 12}}
-        @retentionMonths={{@model.config.retentionMonths}}
-      />
-    {{/if}}
-    {{#if @isLoading}}
-      <LayoutLoading />
-    {{else}}
-      {{#if this.hasClientData}}
-        {{! ARG TODO end of part that goes to Running Client }}
-        {{#if this.showGraphs}}
-          {{! ARG TODO chart playground }}
-          <Clients::RunningTotal
-            @title="Vault client counts"
-            @description="An active client is any user or service that interacts with Vault. They are made up of unique entities and non-entity tokens. The total client count number is an important consideration for Vault billing."
-            @chartLegend={{this.chartLegend}}
-            @lineChartData={{this.lineChartData}}
-            @barChartData={{this.newMonthlyClients}}
-          />
+      {{! check for startTime otherwise emptyState}}
+      {{#if @model.startTime}}
+        <div class="calendar-title">Filters</div>
+        <Toolbar>
+          <ToolbarFilters>
+            <CalendarWidget
+              @startDate={{this.startDate}}
+              @handleEndMonth={{this.handleEndMonth}}
+              @endDate={{@model.newInitActivity.endTime}}
+            />
+            {{! ARG TODO more filters for auth and namespace here }}
+          </ToolbarFilters>
+        </Toolbar>
+        {{#if @isLoading}}
+          <LayoutLoading />
+        {{else}}
+          {{#if this.hasClientData}}
```

Comment:
Is the nested `#if` block needed or could you do `{{else if this.hasClientData}}` instead?

Reply:
Good call. A little cleaner for sure.